src/commonMain/kotlin/fr/acinq/lightning/NodeParams.kt

```diff
@@ -136,30 +132,22 @@ data class NodeParams(
     val maxAcceptedHtlcs: Int,
     val expiryDeltaBlocks: CltvExpiryDelta,
     val fulfillSafetyBeforeTimeoutBlocks: CltvExpiryDelta,
-    val checkHtlcTimeoutAfterStartupDelaySeconds: Int,
+    val checkHtlcTimeoutAfterStartupDelay: Duration,
+    val checkHtlcTimeoutInterval: Duration,
     val htlcMinimum: MilliSatoshi,
     val toRemoteDelayBlocks: CltvExpiryDelta,
     val maxToLocalDelayBlocks: CltvExpiryDelta,
     val minDepthBlocks: Int,
     val feeBase: MilliSatoshi,
-    val feeProportionalMillionth: Int,
-    val revocationTimeoutSeconds: Long,
-    val authTimeoutSeconds: Long,
-    val initTimeoutSeconds: Long,
-    val pingIntervalSeconds: Long,
-    val pingTimeoutSeconds: Long,
-    val pingDisconnect: Boolean,
-    val autoReconnect: Boolean,
-    val initialRandomReconnectDelaySeconds: Long,
-    val maxReconnectIntervalSeconds: Long,
-    val chain: Bitcoin.Chain,
-    val channelFlags: Byte,
-    val paymentRequestExpirySeconds: Long,
-    val multiPartPaymentExpirySeconds: Long,
+    val feeProportionalMillionths: Int,
+    val pingInterval: Duration,
+    val initialRandomReconnectDelay: Duration,
+    val maxReconnectInterval: Duration,
+    val paymentRequestExpirySeconds: Int,
```

Comment:
`paymentRequestExpirySeconds` stays untouched as we need an exact value in seconds.

Reply:
It was actually unused, removed in 7447a5cb.